lib/arel/visitors/to_sql.rb

```diff
@@ -166,6 +166,28 @@ def visit_Arel_Nodes_False o, collector
         collector << "FALSE"
       end
 
+      def visit_Arel_Nodes_Tuple o, collector
+        collector << "VALUES "
+
+        len = o.values.length - 1
+        o.values.each_with_index { |value, i|
+          collector << '('
+          case value
```

Comment:
This `case` (specifically, the check for BindParam) should be applied to the array elements